README.md

```diff
@@ -27,7 +27,9 @@ focused in scope while avoiding unrelated commits and changes.
 - Push your branch `git push origin cool.new.feature`
 - Submit a [pull request](https://github.com/VivyTeam/vivy-components/pulls) and add details about your changes
 - Add a reviewer (or ask someone in person) to go over your changes
-- If all goes well click the "Squash and Merge" button to close your PR (make sure to clean up and delete your branch)
+- When the PR is approved click the "Squash and Merge" button to close your PR (make sure to clean up and delete your branch)
+- Create a new release by following previous releases format and pick a relevant version according to the change (patch, minor, major)
```

Comment:
maybe mention github clearly or even better add this link https://github.com/vivyteam/vivy-components/releases